presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java

```diff
@@ -757,12 +757,12 @@ public void testComputePartitionStatisticsOnCreateTable()
                     row("c_tinyint", null, 1.0, 0.5, null, "120", "120"),
                     row("c_smallint", null, 1.0, 0.5, null, "32760", "32760"),
                     row("c_int", null, 1.0, 0.5, null, "2147483640", "2147483640"),
-                    row("c_bigint", null, 1.0, 0.5, null, "9223372036854775800", "9223372036854775800"),
+                    row("c_bigint", null, 1.0, 0.5, null, "9223372036854775807", "9223372036854775807"),
```

Comment:
why it differs?

Reply:
This value is out of range of the values, precisely represented by double. Before, when the value was transferred as `Object` it was truncated when converting to `SymbolStatsEstimate`. But now the truncation is visible to the end user.